app.json

```diff
@@ -45,12 +45,14 @@
     "MOBILECOMMONS_COMPANY_KEY": {
       "required": true
     },
-    // @see https://github.com/DoSomething/ds-mdata-responder/issues/518 
     "NEW_RELIC_APP_NAME": {
-      "required": false
+      "required": true
+    },
+    "NEW_RELIC_LICENSE_KEY": {
+      "required": true
     },
-    "NEWRELIC_LICENSE_KEY": {
```

Comment:
This is an old  Nodejitsu environment variable we should purge: https://github.com/DoSomething/ds-mdata-responder/commit/6c5fe842deb1a9fa2247b30d5a6e90835942daeb